view/frontend/templates/vat-id/validation-status-js.phtml

```diff
@@ -81,6 +85,10 @@
 
                     console.debug(`Start VAT validation ${this.vatNumber} for country ${this.countryId}`);
```

Comment:
@Tjitse-E please, remove `console.debug` line here and https://github.com/Vendic/hyva-checkout-geissweb-euvat/blob/b28fbca0842337bb7f4906458a0f8cec98c31b45/view/frontend/templates/vat-id/validation-status-js.phtml#L110